src/test/regress/expected/multi_extension.out

```diff
@@ -1045,6 +1045,22 @@ SELECT * FROM multi_extension.print_extension_changes();
 -- Test downgrade script (result should be empty)
 ALTER EXTENSION citus UPDATE TO '11.0-1';
 ALTER EXTENSION citus UPDATE TO '11.0-2';
+ALTER EXTENSION citus UPDATE TO '11.0-3';
+-- Snapshot of state at 11.0-3
+SELECT * FROM multi_extension.print_extension_changes();
+ previous_object | current_object
+---------------------------------------------------------------------
+(0 rows)
+
+-- Test downgrade script (result should be empty)
+ALTER EXTENSION citus UPDATE TO '11.0-2';
+SELECT * FROM multi_extension.print_extension_changes();
+ previous_object | current_object
+---------------------------------------------------------------------
+(0 rows)
+
+-- get back to the latest version
+ALTER EXTENSION citus UPDATE TO '11.0-3';
```

Comment:
Hmm., why does:
```
-- show running version
SHOW citus.version;
 citus.version
---------------------------------------------------------------------
 11.0.2
(1 row)
```
in 5 lines later?

Reply:
Do we change those when we prepare for the rerelease? https://github.com/citusdata/citus/commit/6663635593ed70a50f98f4688b9fb8ccfa0662d0

Reply:
Yes. We update that when we bump the versions in configure scripts.

Notice that this version is not the migration version. The formatting is slightly different.